db/models/review.js

```diff
@@ -1,7 +1,3 @@
-'use strict'
-const Sequelize = require('sequelize')
-const db = require('./index.js')
```

Comment:
Lines 1 and 2 are still necessary or there will be errors, it's just db that isn't used...